R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
+}
+
+set_session_id <- function() {
+  if (is.null(.get_safely_from_envir("session_id"))) {
+    if (requireNamespace("uuid", quietly = TRUE)) {
+      assign("session_id", uuid::UUIDgenerate(), envir = .lockbox_env)
```

Comment:
Does `.lockbox_env$session_id <- .lockbox_env$session_id %||% uuid::UUIDgenerate()` operate differently? I was thinking these may be the same while also avoiding the outside `if` null check. It would also be consistent with other